src/start_server.js

```diff
@@ -63,7 +64,11 @@ function startWithPort(options) {
     res.redirect(301, `/components/${polyserve.packageName}/`);
   });
 
-  app.use('/components/', polyserve);
+  if (options.cors) {
+    app.use('/components/', cors(), polyserve);
```

Comment:
sgtm then, I'd just never used it and always just added headers myself. Maybe we can get rid of the headers parameter to makeApp as well.